### PR TITLE
Add a python server with hello world

### PR DESCRIPTION
**Pull request by benstest**

### CODE REVIEW

File: server.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import cherrypy
```
Comment (andrewjdmccann) [inline server.py:1]: What is this black magic!? Where are the semi-colons!!?

File: server.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+	  
+class HelloWorld(object):
+    def index(self):
+        return "Hello World!"
+    index.exposed = True
```
Comment (benstest) [inline server.py:6]: Looks like there's a bug.

Comment (benstest) [inline server.py:6]: No, you're wrong.

File: server.py [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import cherrypy
+	  
+class HelloWorld(object):
+    def index(self):
```
Comment (benstest) [inline server.py:4]: Some comments.

**Pull request merged**